client/components/domains/register-domain-step/style.scss

```
@@ -118,6 +119,22 @@
 	}
 }
 
+button.register-domain-step__next-page-button {
+	font-size: 0.875rem;
+	font-weight: 500; /* stylelint-disable-line */
+}
+
+
+body.is-section-domains {
+	.register-domain-step {
+		button.register-domain-step__next-page-button {
```

Is the `button` selector necessary in here? 
If not, then it could be simplified using the parent selector: 

```suggestion
	.register-domain-step {
		&__next-page-button {
```

---

Good point...and the one below too.

---

Eh. Actually I've decided to revert this change since it's more searchable with the full selector string.